Assets/HoloToolkit/Build/Editor/BuildDeployWindow.cs

```diff
@@ -0,0 +1,581 @@
+                    devicePassword = newPassword;
+                    EditorPrefs.SetBool(EditorPrefs_FullReinstall, newFullReinstall);
+                    fullReinstall = newFullReinstall;
+                }
+
+                // Build list (with install buttons)
+                if (this.builds.Count == 0)
+                {
+                    GUILayout.Label(kGUIHorizSpacer + "*** No builds found in build directory", EditorStyles.boldLabel);
+                }
+                else
+                {
+                    foreach (var fullBuildLocation in this.builds)
+                    {
+                        int lastBackslashIndex = fullBuildLocation.LastIndexOf("\\");
+
+                        var directoryDate = Directory.GetLastWriteTime(fullBuildLocation).ToString("yyyy/MM/dd HH:mm:ss");
+                        string packageName = fullBuildLocation.Substring(lastBackslashIndex + 1);
+
+                        EditorGUILayout.BeginHorizontal();
+                        GUILayout.Space(kGUISectionOffset + 15);
+                        if (GUILayout.Button("Install", GUILayout.Width(120.0f)))
+                        {
+                            string[] IPlist = ParseIPList(targetIPs);
+                            EditorApplication.delayCall += () =>
+                            {
+                                InstallAppOnDevicesList(fullBuildLocation, appName, fullReinstall, IPlist);
+                            };
+                        }
+                        GUILayout.Space(5);
+                        GUILayout.Label(packageName + " (" + directoryDate + ")");
+                        EditorGUILayout.EndHorizontal();
+                    }
+                    EditorGUILayout.Separator();
+                }
+            GUILayout.EndVertical();
+
+            GUILayout.Space(kGUISectionOffset);
+
+            // Utilities section
+            GUILayout.BeginVertical();
+                GUILayout.Label("Utilities");
+
+                // Open web portal
+                using (new EditorGUILayout.HorizontalScope())
+                {
+                    GUI.enabled = ShouldWebPortalBeEnabled;
+                    GUILayout.FlexibleSpace();
+                    if (GUILayout.Button("Open Device Portal", GUILayout.Width(buttonWidth_Full)))
+                    {
+                        OpenWebPortalForIPs(this.targetIPs);
+                    }
+                    GUI.enabled = true;
+                }
+
+                // Launch app..
+                using (new EditorGUILayout.HorizontalScope())
+                {
+                    GUI.enabled = ShouldLaunchAppBeEnabled;
+                    GUILayout.FlexibleSpace();
+                    if (GUILayout.Button("Launch Application", GUILayout.Width(buttonWidth_Full)))
+                    {
+                        // If already running, kill it (button is a toggle)
+                        if (IsAppRunning_FirstIPCheck(appName, this.targetIPs))
```

Comment:
Hm, is this needed? I don't remember having to kill apps that were running. Thought AppX tooling on the phone handled this for us.